website/docs/getting-started/for-designers/partials/design-language.md

```diff
@@ -0,0 +1 @@
+Something here about the design language
```

Comment:
Possible I'm out of the loop, but to me this seems like something we can add post-launch

Reply:
Referencing [this ticket](https://hashicorp.atlassian.net/browse/HDS-1314?focusedCommentId=170569&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-170569)